lib/profile-fetch.js

```diff
@@ -0,0 +1,32 @@
+module.exports = profileFetch
+
+var get = require('lodash.get')
+var merge = require('lodash.merge')
+var set = require('lodash.set')
+
+var internals = module.exports.internals = {}
+internals.fetchProperties = require('../utils/fetch-properties')
+internals.saveSession = require('../utils/save-session')
```

Comment:
I use the `internals` so we can mock it in our tests, you'll see that later with things like `simple.mock(profileFetch.internals, 'fetProperties').resolveWith(...)`

That we we don’t need to use `nock` to mock http requests, which makes our tests more unit-esque